tests/Type/data/query-builder.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace EloquentBuilder;
+namespace QueryBuilder;
```

Comment:
# 😲

Reply:
Seemed like an error to me, or was I wrong? 😄